README.md

```diff
@@ -23,6 +37,22 @@ For information regarding testing, please go to [testing](docs/testing.md).
 
 For information regarding deployment, please go to [deployment](docs/deployment.md).
 
+### Publishing to npm
```

Comment:
Wozu steht das im READ.md? Sollen user ein npm package veröffentlichen? Wozu?

Reply:
This is under:
Getting started -> ... for development -> publishing to npm

I put it there for myself and the next maintainers to find. But yes, I should move it.

Reply:
moved it